templates/web/smidsy/reports/index.html

```diff
@@ -1,55 +1,145 @@
+        </p>
+        <p>
+            <label for="transport">[% loc('Transport') %]</label>
+            <select class="form-control" id="transport" name="transport">
+                <option value="">All</option>
+                [% FOR p IN participant_labels.pairs %]
+                <option value="[% p.key %]"[% ' selected' IF transport == p.key %]>[% p.value | html %]</option>
+                [% END %]
+            </select>
+        </p>
+        <p class="no-label">
+            <input class="btn" value="[% loc('Update') %]" type="submit">
+        </p>
+    </div>
 </form>
 
-<table cellpadding="3" cellspacing="1" border="0" class="nicetable">
-<thead>
-<tr>
-<th class="title">[% loc('Responsible body') %]</th>
-<th class="data">[% loc('Incidents<br>this week') %]</th>
-<th class="data">[% loc('Near misses<br>this week') %]</th>
-<th class="data">[% loc('Older<br>incidents') %]</th>
-<th class="data">[% loc('Older<br>near misses') %]</th>
-</tr>
-</thead>
+<div class="dashboard-row">
+    <div class="dashboard-item dashboard-item--12">
+        <div class="labelled-line-chart">
+            <canvas id="chart-all-reports" width="600" height="300"
+                data-labels="[&quot;[% problem_periods.join('&quot;,&quot;') %]&quot;]"
+                data-values-reports="[[% problems_reported_by_period.join(',') %]]"
+                ></canvas>
+            <span class="label" data-datasetindex="0"><strong style="color: #F4A140">[% tprintf(nget("%s problem reported", "%s problems reported", problems_reported_by_period.last), decode(problems_reported) _ '</strong>') %]</span>
+        </div>
+    </div>
+</div>
 
-<tbody>
-[% FOREACH body IN bodies %]
-    <tr align="center"
-    [%- IF NOT body.areas.size %] class="gone"
-    [%- ELSIF ! (loop.count % 2) %] class="a"
-    [%- END %]>
-    <td class="title"><a href="[% body.url(c) %]">[% body.name %]</a></td>
-    <td class="data">[% new_accident.${body.id} or 0 %]</td>
-    <td class="data">[% new_miss.${body.id} or 0 %]</td>
-    <td class="data">[% ( older_accident.${body.id} || 0 ) + ( unknown_accident.${body.id} || 0 ) %]</td>
-    <td class="data">[% ( older_miss.${body.id} || 0 ) + ( unknown_miss.${body.id} || 0 ) %]</td>
-    </tr>
-    [% TRY %][% PROCESS "reports/_extras.html" %][% CATCH file %][% END %]
-[% END %]
-</tbody>
-</table>
+<div class="dashboard-row">
+    <div class="dashboard-item dashboard-item--6">
+        <h2 class="dashboard-subheading">[% loc('Download the data on this page') %]</h2>
+        <p>Grab a CSV of all the reports matching the filters you’ve set at the top of this page, suitable for analysis in applications like Excel or Fusion Tables.</p>
+        <a href="[% c.uri_with( {csv => 1} ) %]" class="btn">Download CSV</a>
+    </div>
+    <div class="dashboard-item dashboard-item--6">
+        <form class="dashboard-search">
+            <h2>[% loc('Show reports on a map') %]</h2>
+            <label for="body">[% loc('Pick a council') %]</label>
```

Comment:
Can we make this "Find" instead of "Pick"? As @LouiseMySociety noted, it’s a search box, you can’t "pick" anything until you’ve typed a search term.